src/MetaModels/Attribute/Timestamp/Timestamp.php

```diff
@@ -54,7 +58,7 @@ public function getFieldDefinition($arrOverrides = array())
         $strDateType                       = $this->getDateTimeType();
         $arrFieldDef                       = parent::getFieldDefinition($arrOverrides);
         $arrFieldDef['eval']['rgxp']       = $strDateType;
-        $arrFieldDef['eval']['datepicker'] = ($strDateType !== 'time');
+        $arrFieldDef['eval']['datepicker'] = !empty($arrFieldDef['eval']['readonly'])?false:true;
```

Comment:
is this necessary?
...the attribute timestamp require "contao/core" and this require "contao-components/all" and this require "contao-components/datepicker"